modules/vba_documents/app/views/vba_documents/unsuccessful_report_mailer/unsuccessful_report.html.erb

```diff
@@ -46,7 +46,7 @@
   <tfoot>
   <% if @consumer_totals %>
     <% sum_hash = @consumer_totals['summary'] %>
-    <tr style="background-color: darkblue">
+    <tr style="background-color: #ADD8E6">
```

Comment:
This color is a lighter blue that has a higher contrast with the black text and should therefore be easier on the eyes.